src/components/molecules/TabsContainer.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import TabMenu from './TabMenu';
+
+interface IProps {
+  className?: string;
+  children: Array<JSX.Element>;
+}
+
+interface IPropsTabContent {
+  className?: string;
+  id: string;
+  labelledBy: string;
+  role: string;
+  label: string;
+  activeTab: string;
+}
+
+const TabContent: React.FC<IPropsTabContent> = props => {
+  return (
+    <div
+      className={props.className}
+      id={props.id}
+      role={props.role}
+      aria-labelledby={props.labelledBy}
+    >
+      {props.children}
+    </div>
+  );
+};
+
+const TabsContainer: React.FC<IProps> = props => {
+  const [activeTab, setActiveTab] = React.useState(
+    props.children[0].props.label
+  );
+
+  const tabLabels = new Array(props.children.length);
+  props.children!.forEach(child => {
+    tabLabels.push(child.props.label);
+  });
+
+  return (
+    <div className={props.className}>
+      <TabMenu
+        className="tab-menu"
+        tabLabels={tabLabels}
+        activeTab={activeTab}
+        setActiveTab={setActiveTab}
+      />
+      {props.children
```

Comment:
Gladly accept inputs on better, cleaner, and/or terser ways to render the content based on the activeTab state.